packages/components/tools/templates.js

```diff
@@ -84,6 +84,22 @@ const loadContents = glob =>
   getContents(glob).then(contents => {
     contents.forEach((content, templateName) => {
       Handlebars.registerPartial(templateName, content);
+      Handlebars.registerHelper('isFirstDataTd', function(columns, index) {
+        var firstDataTdIndex = 0;
+        for (var i = 0; i < columns.length; ++i) {
+          if (
+            columns[i].section |
```

Comment:
Did you mean `||`?

Reply:
fixed in https://github.com/carbon-design-system/carbon/pull/2992/commits/5fe1847a7841cd3c474cb413a8ed1b69a7cc618a